app/components/containers/containers.html

```diff
@@ -100,8 +100,8 @@
             </tr>
           </thead>
           <tbody>
-            <tr dir-paginate="container in (state.filteredContainers = ( containers | filter:state.filter | orderBy:sortType:sortReverse | itemsPerPage: state.pagination_count))">
-              <td><input type="checkbox" ng-model="container.Checked" ng-change="selectItem(container)"/></td>
+            <tr ng-click="container.Checked = !container.Checked; selectItem(container);" ng-class="{active: container.Checked}" dir-paginate="container in (state.filteredContainers = ( containers | filter:state.filter | orderBy:sortType:sortReverse | itemsPerPage: state.pagination_count))">
```

Comment:
Could you update the `ng-class` so that the `interactive` class is always added ?

E.g. `ng-class="{ active: container.Checked, interactive: true }"`